src/en/proposals/pda-messaging.md

```diff
@@ -0,0 +1,71 @@
+## PDA messaging program and server
+
+[Julian, VasilisThePikachu, Unapproved]
+
+*(Taken by [Julian doc](https://hackmd.io/iu2yK9bcQb-veuCOLl-FYw?both#Optional-Channels-and-Department-based-Channels) in hackmd and modified a lil. Mostly replacing "email" to "message", "Email address" to "user/user id" and adding some of my own twists. Julian was fine with this if i understood correctly (i was in vc with em))*
+
+*(This is mostly taken from how PDA messages work in ss13)*
+Allows sending messages to others using PDAs
+
```

Comment:
There is a lot of focus on technical details, which is good. But this also deserves some high level discussion of why this is needed in the first place, e.g. why not just use normal radio channels.

It would be helpful to discuss some of the things that were discussed on Discord. If everyone started using PDA messages, folks would lose situational awareness in the general radio channels, no?

Reply:
Can you please explain why you think this will make people "lose situational awareness"

From how i see it i dont see how it would and neither did everyone else on the discord. Can you tell me why you think so?